warehouse/integrations/clickhouse/clickhouse.go

```diff
@@ -85,36 +85,78 @@ var datatypeDefaultValuesMap = map[string]interface{}{
-	"Array(String)":                    "array(string)",
-	"Array(Nullable(String))":          "array(string)",
-	"DateTime":                         "datetime",
-	"Array(DateTime)":                  "array(datetime)",
-	"Array(Nullable(DateTime))":        "array(datetime)",
-	"UInt8":                            "boolean",
-	"Array(UInt8)":                     "array(boolean)",
-	"Array(Nullable(UInt8))":           "array(boolean)",
+	"Int8":     "int",
+	"TINYINT":  "int",
+	"BOOL":     "int",
+	"BOOLEAN":  "int",
+	"INT1":     "int",
+	"Int16":    "int",
+	"SMALLINT": "int",
+	"INT2":     "int",
+	"Int32":    "int",
+	"INT":      "int",
+	"INT4":     "int",
+	"INTEGER":  "int",
+	"Int64":    "int",
+	"BIGINT":   "int",
+	"Int128":   "int",
+	"Int256":   "int",
+
+	"Float32": "float",
+	"FLOAT":   "float",
+	"Float64": "float",
+	"DOUBLE":  "float",
+
+	"String":     "string",
+	"LONGTEXT":   "string",
+	"MEDIUMTEXT": "string",
+	"TINYTEXT":   "string",
+	"TEXT":       "string",
+	"LONGBLOB":   "string",
+	"MEDIUMBLOB": "string",
+	"TINYBLOB":   "string",
+	"BLOB":       "string",
+	"VARCHAR":    "string",
+	"CHAR":       "string",
+	"UUID":       "string",
+
+	"Date":              "datetime",
+	"Date32":            "datetime",
+	"DateTime":          "datetime",
+	"DateTime('UTC')":   "datetime",
+	"DateTime64":        "datetime",
+	"DateTime64('UTC')": "datetime",
+
+	"UInt8": "boolean",
+	"Bool":  "boolean",
+
+	"Array(Int64)":                  "array(int)",
+	"Array(Nullable(Int64))":        "array(int)",
+	"Array(Float64)":                "array(float)",
+	"Array(Nullable(Float64))":      "array(float)",
+	"Array(String)":                 "array(string)",
+	"Array(LowCardinality(String))": "array(string)",
+	"Array(Nullable(String))":       "array(string)",
+	"Array(DateTime)":               "array(datetime)",
+	"Array(Nullable(DateTime))":     "array(datetime)",
+	"Array(UInt8)":                  "array(boolean)",
+	"Array(Nullable(UInt8))":        "array(boolean)",
+
 	"LowCardinality(String)":           "string",
 	"LowCardinality(Nullable(String))": "string",
-	"Nullable(Int8)":                   "int",
-	"Nullable(Int16)":                  "int",
-	"Nullable(Int32)":                  "int",
-	"Nullable(Int64)":                  "int",
-	"Nullable(Float32)":                "float",
-	"Nullable(Float64)":                "float",
-	"Nullable(String)":                 "string",
-	"Nullable(DateTime)":               "datetime",
-	"Nullable(UInt8)":                  "boolean",
+
+	"Nullable(Int8)":            "int",
+	"Nullable(Int16)":           "int",
+	"Nullable(Int32)":           "int",
+	"Nullable(Int64)":           "int",
+	"Nullable(Float32)":         "float",
+	"Nullable(Float64)":         "float",
+	"Nullable(String)":          "string",
+	"Nullable(UUID)":            "string",
+	"Nullable(DateTime)":        "datetime",
+	"Nullable(DateTime('UTC'))": "datetime",
+	"Nullable(UInt8)":           "boolean",
```

Comment:
We have added these datatypes.
TINYINT,  INT1, SMALLINT, INT2, INT, INT4, INTEGER, BIGINT, INT128, INT256, FLOAT, DOUBLE, LONGTEXT, MEDIUMTEXT, TINYTEXT, TEXT, LONGBLOB, MEDIUMBLOB, TINYBLOB, BLOB, VARCHAR, CHAR, UUID, DateTime('UTC'), DateTime64, DateTime64('UTC'), Nullable(UUID), Nullable(DateTime('UTC'))